pkg/maps/lbmap/ipv6.go

```diff
@@ -184,10 +184,10 @@ func NewRevNat6Value(ip net.IP, port uint16) *RevNat6Value {
 }
 
 func (k *RevNat6Value) GetValuePtr() unsafe.Pointer { return unsafe.Pointer(k) }
```

Comment:
I think this one should be `v` instead of `k`, no? (And the following ones would be kept as `v`)